assignments/prototypes.js

```diff
@@ -14,34 +14,41 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
+function GameObject(attribute)  {
+  this.createdAt = attribute.createdAt;
+  this.dimensions = attribute.dimensions;
 
-/*
-  === CharacterStats ===
-  * healthPoints
-  * name
-  * takeDamage() // prototype method -> returns the string '<object name> took damage.'
-  * should inherit destroy() from GameObject's prototype
-*/
 
-/*
-  === Humanoid (Having an appearance or character resembling that of a human.) ===
-  * team
-  * weapons
-  * language
-  * greet() // prototype method -> returns the string '<object name> offers a greeting in <object language>.'
-  * should inherit destroy() from GameObject through CharacterStats
-  * should inherit takeDamage() from CharacterStats
-*/
- 
-/*
-  * Inheritance chain: GameObject -> CharacterStats -> Humanoid
-  * Instances of Humanoid should have all of the same properties as CharacterStats and GameObject.
-  * Instances of CharacterStats should have all of the same properties as GameObject.
-*/
+}
+GameObject.prototype.destroy= function() {
+
+  return `${this.name} was removed froom the game`;
+}
+
+function CharacterStats(cattribute) {
+GameObject.call(this,cattribute )
+this.healthPoints = cattribute.healthPoints;
+this.name = cattribute.name;
+
+}
+CharacterStats.prototype = Object.create(GameObject.prototype);
+CharacterStats.prototype.takeDamage = function () {
+  return `${this.name} Took damage.`;
+};
+
+function Humanoid(hattribute) {
+  CharacterStats.call(this, hattribute);
+
+  this.team = hattribute.team;
+  this.weapons = hattribute.weapons;
+  this.language = hattribute.language;
+} 
+Humanoid.prototype = Object.create(CharacterStats.prototype);
+Humanoid.prototype.greet = function () {
+  return `${this.name} Hello how are you ? ${this.language}`
+}
```

Comment:
this is good but make sure to follow the instructions format or it won't come out right
`${this.name} offers a greeting in ${this.language}.`